lib/matplotlib/axes/_axes.py

```diff
@@ -4624,41 +4624,41 @@ def barbs(self, *args, **kw):
                         positional_parameter_names=["x", "y", "c"])
     @docstring.dedent_interpd
     def fill(self, *args, **kwargs):
-        """
-        Plot filled polygons.
-
-        Call signature::
-
-          fill(*args, **kwargs)
-
-        *args* is a variable length argument, allowing for multiple
-        *x*, *y* pairs with an optional color format string; see
-        :func:`~matplotlib.pyplot.plot` for details on the argument
-        parsing.  For example, to plot a polygon with vertices at *x*,
-        *y* in blue.::
-
-          ax.fill(x,y, 'b' )
-
-        An arbitrary number of *x*, *y*, *color* groups can be specified::
-
-          ax.fill(x1, y1, 'g', x2, y2, 'r')
-
-        Return value is a list of :class:`~matplotlib.patches.Patch`
+        """Plot filled polygons.
+
+	Parameters
+	----------
+        args : a variable length argument, allowing for multiple
+            *x*, *y* pairs with an optional color format string; see
+            :func:`~matplotlib.pyplot.plot` for details on the argument
+            parsing.  For example, to plot a polygon with vertices at *x*,
+            *y* in blue.::
+            ax.fill(x,y, 'b' )
+            An arbitrary number of *x*, *y*, *color* groups can be specified::
+            ax.fill(x1, y1, 'g', x2, y2, 'r')
+
+ 	kwargs : The *closed* kwarg will close the polygon when *True* (default).
+            kwargs control the :class:`~matplotlib.patches.Polygon` properties:
+            %(Polygon)s
+
+
+        Return 
```

Comment:
Though it's best to be consistent, of course, if everyone's been writing it one way.